### PR TITLE
V1.7.0

### PR DESCRIPTION
**Pull request by GutierrezJeremy**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -15,47 +15,54 @@
...
-      "version_requirement": ">= 6.0.0 < 7.0.0"
+      "version_requirement": ">= 7.0.0 < 9.0.0"
     }
   ],
   "operatingsystem_support": [
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
-        "8"
+        "7",
```
Comment (fraenki) [inline metadata.json:28]: This module was never intended for CentOS/RedHat 7, please remove this version.

File: metadata.json [hunk 2]
```diff
@@ -15,47 +15,54 @@
...
+        "20.04",
+        "22.04"
       ]
     },
     {
       "operatingsystem": "FreeBSD",
       "operatingsystemrelease": [
-        "12"
+        "11.4",
+        "12.1"
```
Comment (fraenki) [inline metadata.json:60]: Please only specify the current FreeBSD major release number 13.

**Review by fraenki — CHANGES_REQUESTED** [review]

File: metadata.json (hunk 2 above)
Comment (GutierrezJeremy) [reply metadata.json]: @fraenki done

File: metadata.json (hunk 1 above)
Comment (GutierrezJeremy) [reply metadata.json]: @fraenki done

**Review by fraenki — APPROVED** [review]

**Comment by fraenki** [comment]
Merged, thanks!

@GutierrezJeremy, please go ahead and create the release on Github.

**Pull request merged**